app/lib/actions/pulp/orchestration/repository/remove_units.rb

```diff
@@ -0,0 +1,14 @@
+module Actions
+  module Pulp
+    module Orchestration
+      module Repository
+        class RemoveUnits < Pulp::Abstract
+          def plan(_repository, _smart_proxy, options)
+            options[:repo_id] = repository_id
```

Comment:
I think you meant 'repository.id'  This currently causes a failure when trying to remove units from a pulp2 repo